category-committed.php

```diff
@@ -0,0 +1,330 @@
+      </div>
+      <div class="flex-grid-item flex-item-xxl-12 text-align-center pt-10 pt-l-7 pt-s-12">
+        <h1 class="u-visuallyhidden">Committed</h1>
+        <h3 class="u-visuallyhidden">Would you go to prison for your politics?</h3>
+        <div class="committed-archive__logo"></div>
+        <div class="committed-archive__subtitle "></div>
+      </div>
+    </div>
+  </section>
+</div>
+  <section class="container">
+    <div class="flex-grid-row mt-4 mb-4">
+      <div class="flex-grid-item flex-offset-s-0 flex-item-s-12 flex-offset-l-1 flex-item-l-10 flex-offset-xxl-2 flex-item-xxl-8">
+        <h3 class="mb-4 font-size-12 font-weight-bold">Listen to the trailer:</h3>
+        <iframe width="100%" height="166" scrolling="no" frameborder="no" allow="autoplay" src="https://w.soundcloud.com/player/?url=https%3A//api.soundcloud.com/tracks/1270698601&color=%23ffab70&auto_play=false&hide_related=true&show_comments=false&show_user=false&show_reposts=false&show_teaser=false"></iframe>
+      </div>
+    </div>
+    <div class="flex-grid-row mt-4 mb-5">
+      <div class="flex-grid-item flex-offset-s-0 flex-item-s-12 flex-offset-l-1 flex-item-l-10 flex-offset-xxl-2 flex-item-xxl-8 font-serif committed__serif-large text-paragraph-breaks">
+        <p>Britain is locking up peaceful climate protesters. You’ve probably heard of some of them – they’re the road-blockers, soup-throwers and spray-painters who’ve made headlines around the world with Just Stop Oil.</p>
+        <p>In a new podcast series from Novara Media, we meet the real people behind the front pages. What type of person would risk their freedom for a cause they believe in? What’s it really like in Britain’s overcrowded prisons – and is life on the inside any different for political prisoners? How does jail-time strain the relationships with those they love most? And when you’re facing many years inside, how do you make sense of what you’ve done?</P>
+        <p>Hosted by Rivkah Brown and Clare Hymer, Committed is a podcast about why some people are willing to go to prison for their politics.</p>
+        <p>Over four episodes, we follow a group of Just Stop Oil activists into jail and out again, asking them why they did it – and asking ourselves if we’d be willing to do the same.</p>
+      </div>
+    </div>
+    <div class="flex-grid-row mb-6 mb-s-4">
+      <div class="flex-grid-item flex-item-s-12 flex-item-xxl-12 font-color-white committed-archive-listen__box">
+        <div class="committed-archive__box">
+          <div class="committed-listen-now-title mb-1"></div>
+          <div class="font-weight-bold committed-listen-now__links font-size-13 font-size-s-12">
+            <a href="https://podcasts.apple.com/us/channel/novara-media/id6742787656">Apple Podcasts</a>,
+            <a href="https://open.spotify.com/show/3KtmyPhvQ2FvhwdRNo5I1N?si=a95692ce1d75432d">Spotify</a>,<br/><a href="#">Google Podcasts</a>,
```

Comment:
Realised Google Podcasts doesnt even exist anymore. These urls are unlikely to be the final ones because I think it will be a separate feed, but only 4 episodes so maybe not

Reply:
yep we were in coms about the links but chal has given me the correct ones now. google podcast doesn't exist you're right, rip

Reply:
we are using the main nm feed for now and then going to do a patch release to update to a separate feed or the first episode in a few weeks.